.editorconfig

```diff
@@ -1,6 +1,6 @@
-# top-most EditorConfig file
-root = true
-
-[*]
-indent_style = space
-indent_size = 4
+# top-most EditorConfig file
+root = true
+
+[*]
+indent_style = space
+indent_size = 4
```

Comment:
This whole file shows as changed because I changed the newlines from CR/LF to LF to match all the other files in your repo.